public/index.html

```diff
@@ -12,7 +12,13 @@
 		<footer>
 			<script src="/__/firebase/7.8.1/firebase-app.js"></script>
 			<script src="/__/firebase/7.8.1/firebase-analytics.js"></script>
+			<script src="/__/firebase/7.8.0/firebase-performance.js"></script>
```

Comment:
Out of curiosity is there a reason the versions don't match?

Reply:
Not sure, I copied these straight from the docs.